cloud_level_testing/features/steps/tools.py

```diff
@@ -813,18 +813,18 @@ def create_wait_script(conn_test, testname):
     if "iperf" in conn_test:
         secondary = "iperf"
 
-    script_content = f"""
+    script_content = '''
             #!/bin/bash
             let MAXW=100
             if test ! -f /var/lib/cloud/instance/boot-finished; then sleep 5; sync; fi
-            while test \$MAXW -ge 1; do
-            if type -p "{conn_test}">/dev/null || type -p "{secondary}">/dev/null; then exit 0; fi
+            while test $MAXW -ge 1; do
+            if type -p "%s">/dev/null || type -p "%s">/dev/null; then exit 0; fi
```

Comment:
more "pythonic" way to handle formatting is to use the .format() method

Reply:
I know, normally i use f-strings or .format, but in this particular situation i use the c-style printf formatting to stop the complain python interpreter :-)
I added textwrap.dedent to beautify the created script.